### PR TITLE
UML-1186 - Send service usage stats to Slack on a schedule

### PR DESCRIPTION
**Pull request by andrewpearce-digital**
# Purpose

Help keep the team informed on how usage of the service is working.

Fixes UML-1186

## Approach

Update get_statistics.py to output plaintext stats breakdown
Create a CircleCI job to post stats to the slack channel
Schedule job to run on a schedule to be determined by the team.

**Schedule Examples**
Once a week on a Monday morning,  post statistics since service go-live

## Learning

_Any tips and tricks, blog posts or tools which helped you. Plus anything notable you've discovered about the Use a Lasting Power of Attorney service_

## Checklist

* [ ] I have performed a self-review of my own code
* [ ] I have added relevant logging with appropriate levels to my code
* [ ] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* [ ] I have added tests to prove my work
* [ ] I have added welsh translation tags and updated translation files
* [ ] I have run an accessibility tool on any pages I have made changes to and fixed any issues found
* [ ] The product team have tested these changes

### CODE REVIEW

**Comment by andrewpearce-digital** [comment]
Outputting plaintext stats statement
Taking 1 minute to run locally. Will investigate options for improvements
image

File: service-front/app/src/Common/src/Service/User/UserService.php [hunk 1]
```diff
@@ -69,11 +69,14 @@ public function create(string $email, HiddenString $password): array
...
-            'email'      => new Email($email)
-        ]);
+        $this->logger->notice(
+          'Account with Id {id} created using email hash {email}',
+          [
+              'event_code' => EventCodes::ACCOUNT_CREATED,
+              'id'         => $data['Id'],
+              'email'      => new Email($email)
+        ]
+      );
```
Comment (cooperaj) [inline service-front/app/src/Common/src/Service/User/UserService.php:79]:
```suggestion
        $this->logger->notice(
            'Account with Id {id} created using email hash {email}',
            [
                'event_code' => EventCodes::ACCOUNT_CREATED,
                'id'         => $data['Id'],
                'email'      => new Email($email)
            ]
        );
```

File: terraform/environment/terraform.tfvars.json [hunk 2]
```diff
@@ -56,7 +56,7 @@
       "cookie_expires_view": 1440,
       "google_analytics_id_use": "",
       "google_analytics_id_view": "",
-      "logging_level": 250,
+      "logging_level": 200,
```
Comment (cooperaj) [inline terraform/environment/terraform.tfvars.json:59]:
```suggestion
      "logging_level": 250,
```

File: terraform/environment/terraform.tfvars.json [hunk 3]
```diff
@@ -94,7 +94,7 @@
       "cookie_expires_view": 1440,
       "google_analytics_id_use": "UA-170469426-4",
       "google_analytics_id_view": "UA-170469426-3",
-      "logging_level": 250,
+      "logging_level": 200,
```
Comment (cooperaj) [inline terraform/environment/terraform.tfvars.json:97]:
```suggestion
      "logging_level": 250,
```

**Review by cooperaj — COMMENTED** [review]
With the switch to notice logging you should be able to retain the 250 logging level which will keep things less verbose in cloudwatch

**Comment by andrewpearce-digital** [comment]
A previous change has caused a loss of metrics for a period from November 27th. This is ongoing. While this PR restores the emission of events to be captured as metrics, it will not restore the gap in metrics.

File: scripts/get_statistics/output.json [hunk 4]
```diff
@@ -0,0 +1,52 @@
+{
```
Comment (johnnolan) [inline scripts/get_statistics/output.json:1]: Can I get a little context on why we store this in the code base? Does this get amended and committed after each run?

Comment (andrewpearce-digital) [reply scripts/get_statistics/output.json:1]: nah, it's just an example set to run with the --test option. I intend to remove it later, and it's in no way sensitive.

**Review by johnnolan — APPROVED** [review]

**Pull request merged**